modules/shared/src/main/scala/io/constellationnetwork/security/Hash.scala

```diff
@@ -25,11 +24,19 @@ object hash {
 
   object Hash {
 
-    def hashCodeFromBytes(bytes: Array[Byte]): HashCode =
-      Hashing.sha256().hashBytes(bytes)
-
-    def fromBytes(bytes: Array[Byte]): Hash =
-      Hash(hashCodeFromBytes(bytes).toString)
+    def sha256FromBytes(bytes: Array[Byte]): Array[Byte] = {
+      val md = MessageDigest.getInstance("SHA-256")
+      md.update(bytes)
+      md.digest()
+    }
```

Comment:
I think you should just change the implementation of the `hashCodeFromBytes`, not the fn name. We don't know who/how depends on that.

Reply:
I agree, but you need to remember about compatibility too. We do have external dependencies on our code, so we cannot refactor everything as we like to.

Reply:
Such changes can be added to the release notes.